rbac/management/group/view.py

```diff
@@ -440,6 +444,9 @@ def roles(self, request, uuid=None):
             if serializer.is_valid(raise_exception=True):
                 roles = request.data.pop(ROLES_KEY, [])
             add_roles(group, roles)
+            if group.platform_default:
```

Comment:
Since this logic:
```
if group.platform_default:
    group.system = False
    group.save()
```
is used in three different places, maybe we could break it out into a helper method like `set_system_flag_on_update(group)` or something? Then just call that from each of the three places? The first instance where you check for the group first could just be:
```
if group:
    set_system_flag_on_update(group)
```
and the other places it could just call it directly. That would also allow for that logic to be unit tested if we wanted, and if the logic changed in the future we could just updated it once.